include/exact/ExactMapper.hpp

```diff
@@ -6,7 +6,8 @@
 #ifndef EXACT_MAPPER_hpp
 #define EXACT_MAPPER_hpp
 
-#include "Encodings.hpp"
+#include "Encodings/Encodings.hpp"
+#include "LogicBlock/LogicBlock.hpp"
```

Comment:
These includes can be moved to the source file as they are not needed in the header itself.